lemarche/utils/apis/api_hubspot.py

```diff
@@ -63,7 +68,91 @@ def add_to_contacts_async(
             return api_response
         except ApiException as e:
             if e.status == 409:  # conflict error
-                logger.info(f"User {email} already exist in hubpsot crm")
+                # message error body is like this :
+                # '{"status":"error","message":"Contact already exists. Existing ID: 8853",
+                #    "correlationId":"9dfc8","category":"CONFLICT"}'
+                # so we transforme the message from str to dict to get back the ID from hubspot crm
+                dict_error = eval(e.body)
+                regex_hubspot_contact_id = re.search(r"\d+", dict_error.get("message", ""))
+                if regex_hubspot_contact_id:
+                    hubspot_contact_id = regex_hubspot_contact_id.group()
+                    logger.info(f"User {email} already exist in hubpsot crm")
+                    return SimplePublicObject(id=hubspot_contact_id)
+            else:
+                logger.error("Exception when calling hubspot_api->create: %s\n" % e)
+    else:
+        logger.info("Hubspot: not add contact to the crm (STAGING or TEST environment detected)")
+
+
+def add_user_to_crm(user: User):
+    result = add_to_contacts(
+        email=user.email,
+        company=user.company_name,
+        firstname=user.first_name,
+        lastname=user.last_name,
+        phone=user.phone,
+        website=user.c4_website,
+    )
+    if result and result.id:
+        user.set_hubspot_id(hubspot_contact_id=result.id, with_save=True)
+    return result
+
+
+# @task
+def create_deal_from_tender(tender: Tender):
+    user_id_hubspot = tender.author.hubspot_contact_id
```

Comment:
```suggestion
    tender_author_hubspot_contact_id = tender.author.hubspot_contact_id
```
pour être plus précis :)